webapp/channels/src/components/suggestion/switch_channel_provider.tsx

```diff
@@ -545,7 +549,7 @@ export default class SwitchChannelProvider extends Provider {
         }
 
         if (user.id === currentUserId && displayName) {
-            displayName += (' ' + Utils.localizeMessage({id: 'suggestion.user.isCurrent', defaultMessage: '(you)'}));
+            displayName += (' ' + intl.formatMessage({id: 'suggestion.user.isCurrent', defaultMessage: '(you)'}));
```

Comment:
I don't think this whole if statement is actually needed any more because `SwitchChannelSuggestion` also adds the `(you)` to the channel name when displaying it above

Reply:
No, I don't think so. I thought that the value set for `displayName` here isn't actually being used by the `SwitchChannelSuggestion` since it has its own logic for adding the `(you)` to it, but I honestly can't tell if that's the case.

Sorry for the distraction here. Let's just leave this as-is until we can clean up this whole file since it's near impossible to follow